### PR TITLE
blog: Add 2021-17 weekly report

### PR DESCRIPTION
**Pull request by JinnyYi**
Add 2021-17 weekly report.

### CODE REVIEW

File: blog/2021-04-30-weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Library
+
+### Summer 2021 of Open Source Promotion Plan
+
+Open Source Promotion Plan, a.k.a., OSPP, encourage everyone to pay close attention to open source software and open source community, aiming to cultivate and explore more outstanding developers.
+
+This week we applied to participate in OSSP - Summer 2021 and posed the subject . [@Xuanwo](https://github.com/Xuanwo), [@Prnyself](https://github.com/Prnyself) and [@xxchan](https://github.com/xxchan) will take part in the subject, and we will recruit contributors from student groups.
+
+For details, please refer to https://hackmd.io/XcdKgrqFRs6b1CazTxYX7g.
```
Comment (Xuanwo) [inline blog/2021-04-30-weekly-report.md:18]: Use our docs link instead: https://aos.dev/community/events/ospp-summer-2021

File: blog/2021-04-30-weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+It should be noted that the current `delete` behavior should be different for fs and object storage services to make sure all services could pass the integration test . Please keep an eye on our track issue: [Make Delete idempotent](https://github.com/aos-dev/go-storage/issues/551).
+
+### Server Side Encryption Support
+
+Server Side Encryption, a.k.a., SSE is a feature to encrypt data on server side, which is used by various applications. 
+
+ [@xxchan](https://github.com/xxchan) was responsible for completing the SSE support for the services this week, including [Implement SSE support](https://github.com/aos-dev/go-storage/issues/523), [SEE-S3 sample code](https://github.com/aos-dev/go-storage-example/blob/master/sse_s3.go) and related documents. [SSE blog](https://github.com/aos-dev/site/blob/master/blog/2021-04-27-sse.md) shows what is SSE and how to use SSE in `go-storage`.
+
+## dm
```
Comment (Xuanwo) [inline blog/2021-04-30-weekly-report.md:34]: dm should under `Application`

File: blog/2021-04-30-weekly-report.md (hunk 1 above)
Comment (Xuanwo) [reply blog/2021-04-30-weekly-report.md:18]: We can use `<https://aos.dev/community/events/ospp-summer-2021>` to insert a link quickly.

File: blog/2021-04-30-weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+author: "JinnyYi"
+author_url: "https://github.com/JinnyYi"
+tags: [updates, weekly-report]
+---
+
+Hello everyone, here is the weekly report for [AOS](https://aos.dev), range from 2021-04-25 to 2021-04-30.
+
+## Library
+
+### Summer 2021 of Open Source Promotion Plan
```
Comment (Prnyself) [inline blog/2021-04-30-weekly-report.md:12]: This should under `Community` maybe?

File: blog/2021-04-30-weekly-report.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Open Source Promotion Plan, a.k.a., OSPP, encourage everyone to pay close attention to open source software and open source community, aiming to cultivate and explore more outstanding developers.
```
Comment (Prnyself) [inline blog/2021-04-30-weekly-report.md:14]: Add link for this `Open Source Promotion Plan` for more information.

File: blog/2021-04-30-weekly-report.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+This week we applied to participate in OSSP - Summer 2021 and posed the subject . [@Xuanwo](https://github.com/Xuanwo), [@Prnyself](https://github.com/Prnyself) and [@xxchan](https://github.com/xxchan) will take part in the subject, and we will recruit contributors from student groups.
+
+For details, please refer to https://hackmd.io/XcdKgrqFRs6b1CazTxYX7g.
+
+### Appender Support
+
+`Appender` is used to append content to the tail of file.
+
+This week, we implement  `Appender` for the services that support this feature. [CommitAppend](https://github.com/aos-dev/specs/blob/master/rfcs/0-commit-append.md) is introduced to finish an append procedure as service like dropbox needs a `close` to mark this appends process has been finished.
```
Comment (Prnyself) [inline blog/2021-04-30-weekly-report.md:24]: remove redundant blank space plz

File: blog/2021-04-30-weekly-report.md (hunk 1 above)
Comment (xxchan) [reply blog/2021-04-30-weekly-report.md:18]: I think using a link without `<>` is also ok?

File: blog/2021-04-30-weekly-report.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+For details, please refer to https://hackmd.io/XcdKgrqFRs6b1CazTxYX7g.
+
+### Appender Support
+
+`Appender` is used to append content to the tail of file.
+
+This week, we implement  `Appender` for the services that support this feature. [CommitAppend](https://github.com/aos-dev/specs/blob/master/rfcs/0-commit-append.md) is introduced to finish an append procedure as service like dropbox needs a `close` to mark this appends process has been finished. 
+
+It should be noted that the current `delete` behavior should be different for fs and object storage services to make sure all services could pass the integration test . Please keep an eye on our track issue: [Make Delete idempotent](https://github.com/aos-dev/go-storage/issues/551).
```
Comment (Prnyself) [inline blog/2021-04-30-weekly-report.md:26]: Is this another topic or it belongs to `Appender Support` indeed?

File: blog/2021-04-30-weekly-report.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+### Server Side Encryption Support
+
+Server Side Encryption, a.k.a., SSE is a feature to encrypt data on server side, which is used by various applications. 
+
+ [@xxchan](https://github.com/xxchan) was responsible for completing the SSE support for the services this week, including [Implement SSE support](https://github.com/aos-dev/go-storage/issues/523), [SEE-S3 sample code](https://github.com/aos-dev/go-storage-example/blob/master/sse_s3.go) and related documents. [SSE blog](https://github.com/aos-dev/site/blob/master/blog/2021-04-27-sse.md) shows what is SSE and how to use SSE in `go-storage`.
+
```
Comment (xxchan) [inline blog/2021-04-30-weekly-report.md:33]: `[SSE blog](https://aos.dev/blog/2021/04/27/sse)` blog page link is better

File: blog/2021-04-30-weekly-report.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+Hi-Fi  of `dm` service is under designing. The [Hi-Fi prototype interaction](https://www.figma.com/proto/tZBW1fMDLlcdFpaHJYih9B/Data-Migration-Prototype?page-id=1060%3A51&node-id=1060%3A14951&viewport=3370%2C1130%2C0.125&scaling=min-zoom) is on trial and welcome feedback.
+
+## Community
+
+### Summer 2021 of Open Source Promotion Plan
+
+[Open Source Promotion Plan](https://summer.iscas.ac.cn/), a.k.a., OSPP, encourage everyone to pay close attention to open source software and open source community, aiming to cultivate and explore more outstanding developers.
+
+This week we applied to participate in OSSP - Summer 2021 and posed the subject . [@Xuanwo](https://github.com/Xuanwo), [@Prnyself](https://github.com/Prnyself) and [@xxchan](https://github.com/xxchan) will take part in the subject, and we will recruit contributors from student groups.
```
Comment (xxchan) [inline blog/2021-04-30-weekly-report.md:54]: OSSP - typo

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**